ProcessMaker/Http/Controllers/TaskController.php

```diff
@@ -110,7 +110,7 @@ public function edit(ProcessRequestToken $task, string $preview = '')
             }
 
             UserResourceView::setViewed(Auth::user(), $task);
-
+            $currentUser = Auth::user()->only(['id', 'username', 'fullname', 'firstname', 'lastname', 'avatar']);
```

Comment:
Consider to double check if all the required properties are present, please review: `timezone, datetime_format, datetime_format`

![image](https://github.com/ProcessMaker/processmaker/assets/8028650/6b055fbb-5d86-42a8-8da4-c1e41bf934f8)
https://github.com/search?q=org%3AProcessMaker%20%22Processmaker.user%22&type=code

or consider to use a blocklist to remove attributes that should not be sent to frontend.

Reply:
Properties timezone, datetime_format were added to $currentUser variable to avoid problems calling this properties. However, a double check was made in PM core reviewing the global variable Processmaker.user to be sure all properties are included.